mp/game/momentum/gameinfo.txt

```diff
@@ -20,6 +20,38 @@
         test_hardware 1
     }
 
+    // mount
+    // {
+        // 440
+        // {
+            // "tf"
+            // {
+                // "vpk"	"tf2_misc_dir"
+                // "vpk"	"tf2_sound_misc_dir"
+                // "vpk"	"tf2_sound_vo_english_dir"
+                // "vpk"	"tf2_textures_dir"
+                // "dir"	"download"
+            // }
+        // }
+        // 240
+        // {
+            // "cstrike"
+            // {
+                // "vpk"	"cstrike_pak_dir"
+                // "dir"	"download"
+            // }
+        // }
```

Comment:
Apologies, I meant we can actually use this mounting, and have the code for fallback.